### PR TITLE
Stripe

### PR DESCRIPTION
**Pull request by olofvi**
## PT:https://www.pivotaltracker.com/story/show/152641850
-----
* implementing Stripe on front end
* pop up for payment
* payments service for sending necessary params to the backend to create payment
screen shot 2017-11-15 at 10 31 10

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -16,6 +16,9 @@
                   {{item.product_name}} ({{item.price}} kr) x{{item.item_quantity}}
                 </div>
               </div>
+              <div class="payment">
+                <button (click)="openCheckout()">Purchase</button>
```
Comment (Sajaas) [inline src/app/app.component.html]: It should be in Swedish ;)

**Review by Sajaas — APPROVED** [review]
Just one small detail. Otherwise, good job! Magnus and I approve :)

File: src/app/app.component.html (hunk 1 above)
Comment (olofvi) [reply src/app/app.component.html]: please send me the swedish word ;)

**Review by diraulo — APPROVED** [review]
Wanted to have an attempt at testing the payment service, but will do that later.

File: src/app/service/datastore.service.ts [hunk 2]
```diff
@@ -2,12 +2,14 @@ import { Injectable } from '@angular/core';
 import { JsonApiDatastoreConfig, JsonApiDatastore, DatastoreConfig } from 'angular2-jsonapi';
 import { Http } from '@angular/http';
 import { Product } from '../models/product';
+import { Payment} from "../models/payment";
 
 const config: DatastoreConfig = {
   baseUrl: 'http://localhost:3000/api',
```
Comment (diraulo) [inline src/app/service/datastore.service.ts:8]: btw don't you have an API deployed to Heroku already? why not use it here? 🤔

**Pull request merged**